x-pack/plugins/cloud_security_posture/public/pages/findings/findings_by_resource_table.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import {
+  EuiTableFieldDataColumnType,
+  EuiEmptyPrompt,
+  EuiBasicTable,
+  EuiTextColor,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiToolTip,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import numeral from '@elastic/numeral';
+import { extractErrorMessage } from '../../../common/utils/helpers';
+import * as TEST_SUBJECTS from './test_subjects';
+import * as TEXT from './translations';
+import type { CspFindingsByResourceResult } from './use_findings_by_resource';
+
+const formatNumber = (value: number) => (value < 1000 ? value : numeral(value).format('0.0a'));
+
+type FindingsGroupByResourceProps = CspFindingsByResourceResult;
+type CspFindingsByResource = NonNullable<CspFindingsByResourceResult['data']>['page'][number];
+
+const FindingsByResourceTableComponent = ({
+  error,
+  data,
+  loading,
+}: FindingsGroupByResourceProps) => {
+  const getRowProps = () => ({
+    'data-test-subj': TEST_SUBJECTS.FINDINGS_BY_RESOURCE_TABLE_ROW,
+  });
+
+  if (!loading && !data?.page.length)
+    return <EuiEmptyPrompt iconType="logoKibana" title={<h2>{TEXT.NO_FINDINGS}</h2>} />;
+
+  return (
+    <EuiBasicTable
+      loading={loading}
+      error={error ? extractErrorMessage(error) : undefined}
+      items={data?.page || []}
+      columns={columns}
+      rowProps={getRowProps}
+    />
+  );
+};
+
+const columns: Array<EuiTableFieldDataColumnType<CspFindingsByResource>> = [
+  {
+    field: 'resource_id',
+    truncateText: true,
+    name: (
+      <FormattedMessage
+        id="xpack.csp.findings.groupByResourceTable.resourceIdColumnLabel"
+        defaultMessage="Resource Id"
+      />
+    ),
+    render: (resourceId: CspFindingsByResource['resource_id']) => (
+      <EuiToolTip position="top" content={resourceId}>
```

Comment:
Why did you add a tooltip for this column?

Reply:
mainly because columns were copy-pasted from `findings_table` 
but the idea is that the value is a `guid` so it's kinda long and it gets trimmed on lower screens so you can't see it in full. ideally we'd only show the tooltip when it's trimmed. 

we should probably show a short version in all `guid` columns

do you think it should be removed?

Reply:
As I see it this column contains the most important piece of information in the table, and should never be truncated. According to [EUI guidelines](https://elastic.github.io/eui/#/display/tooltip) tooltips should provide short, non-essential, contextual information, which is not the case here

Reply:
that makes more sense! cool, i'll remove the tooltip and truncation from this column. thanks 👍